NineChronicles.Headless/NineChroniclesNodeService.cs

```diff
@@ -277,8 +278,7 @@ internal void ConfigureContext(StandaloneContext standaloneContext)
             standaloneContext.Store = Store;
             standaloneContext.Swarm = Swarm;
             standaloneContext.CurrencyFactory =
-                new CurrencyFactory(
-                    () => standaloneContext.BlockChain.GetAccountState(standaloneContext.BlockChain.Tip.Hash));
+                new CurrencyFactory(() => standaloneContext.BlockChain.GetAccountState((BlockHash?)null));
```

Comment:
I wonder about this change's intention 🤔. It seems the `CurrencyFactory` cannot access any states as my understanding. `BlockChain.GetAccountState -> BlockChainStates.GetAccountState -> BlockChainStates.GetTrie -> TrieStateStore.GetStateRoot`. Is there another change that I missed?

Reply:
Maybe it is just a mistake I guess..?